sklearn/ensemble/_hist_gradient_boosting/gradient_boosting.py

```diff
@@ -1457,6 +1459,20 @@ class HistGradientBoostingClassifier(ClassifierMixin, BaseHistGradientBoosting):
 
         .. versionadded:: 0.24
 
+    See Also
+    --------
+    GradientBoostingClassifier : Implementation of Gradient Boosting for
+        classification with poor performance in big datasets.
```

Comment:
```suggestion
    GradientBoostingClassifier : Exact gradient boosting method
        that does scale as good on datasets with a large number of
        samples.
```

Reply:
`performance` is always tricky because it can refer to the metric or to the computational cost.

Reply:
@glemaitre, I applied your suggestion. There was a typo (a missing "not") so I had to make another commit instead of committing your suggestion directly.